Testscripts/Linux/ntttcp_monitor_throughput.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+#
+# In this script, we want to monitor the network throughput by ntttcp for running very long time, e.g. one month.
+# Any throughput churn over by value provided is logged as Failed.
+
+# Source utils.sh
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    echo "TestAborted" > state.txt
+    exit 0
+}
+# Source constants file and initialize most common variables
+UtilsInit
+
+# Build and install ntttcp
+function build_ntttcp_with_patch () {
```

Comment:
why do we need the patch? why not submit the patch to ntttcp repo?

Reply:
if we do need this, how about moving this function to utils.sh?

Reply:
the patch is used to make the real time output been redirected to file, seems it's a bug of ntttcp for now, not sure if it will be accept or not. discussing about the patch or not is another effort. Once the discussion about ntttcp patch is done, we can make the change to this test script, either remove the patch option, or moving this function to utils.sh.

Reply:
@simonxiaoss Any comment for this change? Should we submit the patch to ntttcp or combine it to the existing install_ntttcp?